stereo_logger:
  ros__parameters:

    subscribers:
      camera_left: /camera_left/raw_frame
      camera_right: /camera_right/raw_frame

    storage_folder: /home/marco/xarm_devel/src/ROS2-Stereo-Logger/saved_images/

    hz_store_image: 0.5